exercises/hello-world/description.md

```diff
@@ -1,46 +1,11 @@
 ["Hello, World!"](http://en.wikipedia.org/wiki/%22Hello,_world!%22_program) is
-the traditional first program for beginning programming in a new language.
+the traditional first program for beginning programming in a new language
+or environment.
 
-**Note:** You can skip this exercise by running:
+The objectives are simple:
 
-    exercism skip $TRACK_ID hello-world
+- Write a function that says "Hello, World!".
+- Run the tests and make sure that they succeed.
+- Submit your solution and check it at the website.
 
-## Specification
-
-Write a `Hello World!` function that can greet someone given their name.  The
-function should return the appropriate greeting.
-
-For an input of "Alice", the response should be "Hello, Alice!".
-
-If a name is not given, the response should be "Hello, World!"
-
-## Test-Driven Development
-
-As programmers mature, they eventually want to test their code.
-
-Here at Exercism we simulate [Test-Driven
-Development](http://en.wikipedia.org/wiki/Test-driven_development) (TDD), where
-you write your tests before writing any functionality. The simulation comes in
-the form of a pre-written test suite, which will signal that you have solved
-the problem.
-
-It will also provide you with a safety net to explore other solutions without
-breaking the functionality.
-
-### A typical TDD workflow on Exercism:
-
-1. Run the test file and pick one test that's failing.
-2. Write some code to fix the test you picked.
-3. Re-run the tests to confirm the test is now passing.
-4. Repeat from step 1.
-5. Submit your solution (`exercism submit /path/to/file`)
-
-## Instructions
-
-Submissions are encouraged to be general, within reason. Having said that, it's
-also important not to over-engineer a solution.
-
-It's important to remember that the goal is to make code as expressive and
-readable as we can. However, solutions to the hello-world exercise will not be
-reviewed by a person, but by rikki- the robot, who will offer an encouraging
-word.
+If everything goes well, you will be ready to fetch your first real exercise.
```

Comment:
Very nice.  One note (probably to myself): while the instructions we lose in this Pull seem appropriate to remove here, they might be useful somewhere else.  If we do end up choosing a `intro-to-tdd` exercise, this might be a good source for that `description.md`.